include/cctz/civil_time_detail.h

```diff
@@ -412,13 +406,17 @@ class civil_time {
 
   // Binary arithmetic operators.
   friend CONSTEXPR_F civil_time operator+(civil_time a, diff_t n) noexcept {
-    return a += n;
+    return civil_time(step(T{}, a.f_, n));
   }
   friend CONSTEXPR_F civil_time operator+(diff_t n, civil_time a) noexcept {
-    return a += n;
+    return a + n;
   }
   friend CONSTEXPR_F civil_time operator-(civil_time a, diff_t n) noexcept {
-    return a -= n;
+    if (n != (std::numeric_limits<diff_t>::min)()) {
+      return civil_time(step(T{}, a.f_, -n));
+    } else {
```

Comment:
We eschew `else` after `return`.  Care to reformulate?

Reply:
Done